services/subnet_manager/src/main/java/com/futurewei/alcor/subnet/service/implement/SubnetServiceImp.java

```diff
@@ -319,6 +318,21 @@ public boolean verifyCidrBlock(String cidr) throws FallbackException {
 
     }
 
+    @Override
+    @DurationStatistics
+    public Integer getUsedIpByRangeId(String rangeId) {
+        String ipManagerServiceUrl = ipUrl + "range" + "/" + rangeId;
+        IpAddrRangeRequest ipAddrRangeRequest = restTemplate.getForObject(ipManagerServiceUrl, IpAddrRangeRequest.class);
```

Comment:
I think the variable shouldn't be called "request" as it is actual a result (I know the class name is confusing).

Recommend to rename the variable to "ipAddrRange".

Reply:
got it